spec/ic.did

```diff
@@ -58,6 +58,8 @@ service ic : {
       module_hash: opt blob;
       memory_size: nat;
       cycles: nat;
+      freezing_threshold: nat;
+      freezing_threshold_cycles: nat;
```

Comment:
Wdyt about adding `in` to denotes the units: `freezing_threshold_in_cycles`

Reply:
Yes, that is fine with me. I will change it to your proposed naming.